lib/OpenQA/Utils.pm

```diff
@@ -903,4 +903,6 @@ sub fix_top_level_help { @ARGV = () if ($ARGV[0] // '') =~ qr/^(-h|(--)?help)$/
 
 sub looks_like_url_with_scheme { return !!Mojo::URL->new(shift)->scheme }
 
+
+sub indent_log_raw_output { @ARGV = () if ($ARGV[0] // '') =~ s/\r\n/,\s\s/gm;}
```

Comment:
`@ARGV`? that's the array with command line arguments to the script

Reply:
Even with `@_`, this doesn't work at all.
i wonder if it even compiles for you.
`\s` on the right hand side of a substitution should generate a warning:
```
Unrecognized escape \s passed through
```
Trying to modify a constant string `... // ''` results in an error here:
```
Can't modify defined or (//) in substitution (s///)
```